packages/react-magma-dom/src/components/Input/Input.stories.tsx

```diff
@@ -304,3 +306,105 @@ NumberInput.args = {
 NumberInput.parameters = {
   controls: { exclude: ['type', 'iconPosition', 'labelWidth'] },
 };
+
+export const SeveralErrors = () => {
+  const [inputValues, setInputValues] = React.useState({
+    first: '',
+    second: '',
+    third: '',
+  });
+  const [hasErrors, setHasErrors] = React.useState({
+    first: true,
+    second: true,
+    third: true,
+  });
+
+  const firstInputRef = React.useRef();
+  const secondInputRef = React.useRef();
+  const thirdInputRef = React.useRef();
+
+  const submit = () => {
+    setHasErrors({
+      first: false,
+      second: false,
+      third: false,
+    });
+
+    if (!inputValues.third) {
+      setHasErrors(prev => ({ ...prev, third: true }));
+      thirdInputRef.current.focus();
+    }
+
+    if (!inputValues.second) {
+      setHasErrors(prev => ({ ...prev, second: true }));
+      secondInputRef.current.focus();
+    }
+
+    if (!inputValues.first) {
+      setHasErrors(prev => ({ ...prev, first: true }));
+      firstInputRef.current.focus();
+    }
+  };
+
+  const reset = () => {
+    setHasErrors({
+      first: false,
+      second: false,
+      third: false,
+    });
+    setInputValues({
+      first: '',
+      second: '',
+      third: '',
+    });
+
+    firstInputRef.current.focus();
+  };
+
+  return (
+    <>
+      <Input
```

Comment:
For the sake of readability (and testing with screen readers, can we update these to be `First name`, `Last name` and `Email address` for the labels, and then make the error messages match ("Please enter your ${variable}")